kafka_2.10-0.10.1.1/pom.xml

```diff
@@ -51,7 +51,7 @@
         </servicemix.osgi.export.pkg>
         <servicemix.osgi.import.pkg>
             com.yammer.metrics*;version="[2.2,3)",
-            org.apache.kafka*;version="[0.9.0.1,0.10.1.0]",
+            org.apache.kafka*;version="[0.10.1.0,0.11.2.0)",
```

Comment:
Note: To use the same version range strategy of previous versions we should set the range to [0.9.0.1,0.11.1.1]